regolith/helpers/a_presentationhelper.py

```diff
@@ -98,6 +100,9 @@ def subparser(subpi):
     subpi.add_argument("--no_cal",
                        help=f"Do not add the presentation to google calendar",
                        action="store_true")
+    subpi.add_argument("--create_talk_repo",
+                       help=f"add presentation to gitlab repo under talks",
+                       action="store_true")
```

Comment:
I would rather that the default be true, so turn off if you don't want a talk repo to be create.  Better yet, if the talk url is in the rc then do it, but if it isn't then don't but print an informational message?  What do you think?